akka-cluster-typed/src/test/scala/akka/cluster/typed/internal/receptionist/ClusterReceptionistSpec.scala

```diff
@@ -41,6 +40,7 @@ object ClusterReceptionistSpec {
         }
       }
       akka.remote.artery.enabled = true
+      akka.remote.artery.transport = tcp
```

Comment:
why? I think it's confusing to understand what is run by our jenkins jobs if we start to do like this
(all cluster tests should be able to run with classic, artery tcp and artery aearon, unless there is a specific reason)

Reply:
Yeap, added it to get less noisy logs when trying to create the test, I'll remove

Reply:
hm, I see that we also have
```
akka.remote.artery.enabled = true
```
in several tests in `akka-cluster-typed` and `akka-cluster-sharding-typed`

That should also be changed. Testing cluster with Artery should be done via jenkins jobs https://jenkins.akka.io:8498/job/akka-artery-cluster-tests/ and https://jenkins.akka.io:8498/job/akka-artery-tcp-cluster-tests/
I have added them there now, because that was missing.